src/app/interfaces/keyboardRows.ts

```diff
@@ -1,4 +1,4 @@
-export const firstKeyBoardRow = [
+export let firstKeyBoardRow = [
```

Comment:
This should be a const, the first time u load the component, make a copy of this object